<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HMIDialog</class>
 <widget class="QDialog" name="HMIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HMI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 237, 204);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>251</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Signal</string>
      </property>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>221</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>BattCoolTagtTemp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>BattHeatTagtTemp</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>PTCActualPwr</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>CompActualPwr</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Reverse</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Reverse</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButtonSigStart">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButtonSigStop">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>VehicleSpeed</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditVSpd">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditBattCoolTemp">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditBattHeatTemp">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditPTCPwr">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditCompPwr">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="title">
       <string>AC</string>
      </property>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>20</width>
         <height>541</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_10">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>331</width>
         <height>64</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushButtonON">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
          </property>
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButtonAUTO">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
            </property>
            <property name="text">
             <string>AUTO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAC">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
            </property>
            <property name="text">
             <string>AC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonREC">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
            </property>
            <property name="text">
             <string>REC</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_11">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>440</y>
         <width>649</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="pushButtonLTempSub">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTempL">
          <property name="text">
           <string>22</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLTempPlus">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDual">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Dual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRTempSub">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTempR">
          <property name="text">
           <string>22</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRTempPlus">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>150</y>
         <width>290</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QPushButton" name="pushButtonBLSub">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBLVal">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBLPlus">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonDEF">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>140</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
       </property>
       <property name="text">
        <string>DEF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonACStop">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>20</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 12pt &quot;Arial&quot;;
border-style: outset;</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonACStart">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 10pt &quot;Arial&quot;;
border-style: outset;</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>761</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>138</y>
         <width>95</width>
         <height>256</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonEn">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>22.0
——
22.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLDef">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>DEF

</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLAuto">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>AUTO

</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_14">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>62</width>
         <height>196</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButtonWindow">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 8pt &quot;Arial&quot;;
border-style: outset;</string>
          </property>
          <property name="text">
           <string>Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFace">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 10pt &quot;Arial&quot;;
border-style: outset;</string>
          </property>
          <property name="text">
           <string>Face</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFoot">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 30px;  border: 2px groove gray;
font: 10pt &quot;Arial&quot;;
border-style: outset;</string>
          </property>
          <property name="text">
           <string>Foot</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
